drivers/timers/arch_alarm.c

```diff
@@ -102,9 +102,9 @@ static void udelay_coarse(useconds_t microseconds)
 static void oneshot_callback(FAR struct oneshot_lowerhalf_s *lower,
                              FAR void *arg)
 {
-  clock_t now;
-  ONESHOT_TICK_CURRENT(g_oneshot_lower, &now);
+  clock_t now = 0;
 
+  ONESHOT_TICK_CURRENT(g_oneshot_lower, &now);
```

Comment:
I think this modification is not necessary, since the ONESHOT_TICK_CURRENT will set it with the initial value. I think we did it just to fix the compiler warning, right?

Reply:
No, the value doesn't set if the driver doesn't implement the related callback after https://github.com/apache/nuttx/pull/14619.